### PR TITLE
U ARen'T Reading the Documentation

### PR DESCRIPTION
**Pull request by mjmagee991**
Enabled UART pin swapping

### CODE REVIEW

**Comment by jlb1854** [comment]
I'm glad we have the hardware in place to fix this issue. Thank you for implementing this software feature.

**Review by jlb1854 — APPROVED** [review]
Great work!

**Review by jlb1854 — CHANGES_REQUESTED** [review]
Is this the best implementation you can have? This may cause confusion as it affects the entire repository, can you pass this as an argument to the constructor in the same manner as the pins? You could also implement this as part of the switch case that determines which halUart object you are referencing with the pins passed to the constructor.

File: include/EVT/io/platform/f3xx/UARTf3xx.hpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ class UARTf3xx : public UART {
      * @param[in] rxPin THe UART RX pin.
      * @param[in] baudrate The baudrate to operate the UART with/
      */
```
Comment (ActuallyZach) [inline include/EVT/io/platform/f3xx/UARTf3xx.hpp:21]: What does isSwapped do? There should be documentation for it.

**Review by ActuallyZach — CHANGES_REQUESTED** [review]

**Review by ActuallyZach — APPROVED** [review]
Looks good!

**Comment by jlb1854** [comment]
Great job!

**Review by jlb1854 — APPROVED** [review]

**Pull request merged**